.cspell/cspell.json

```
@@ -32,6 +32,10 @@
     "/core/includes/transliteration/",
     "/core/misc/ckeditor/",
     "/core/misc/jquery.js",
+    "/core/misc/jquery-extend*",
+    "/core/misc/jquery-html-prefilter*",
+    "/core/misc/jquery.color.js",
+    "/core/misc/jquery.*.js",
```

Don't really need both of these here. The latter covers the former, so that one can go

```suggestion
    "/core/misc/jquery.*.js",
```